File: support_tools/swigwin-3.0.12/Examples/test-suite/go/varargs_overload_runme.go
package main

import "./varargs_overload"

func main() {
	if varargs_overload.Vararg_over1("Hello") != "Hello" {
		panic(0)
	}
	if varargs_overload.Vararg_over1(2) != "2" {
		panic(1)
	}

	if varargs_overload.Vararg_over2("Hello") != "Hello" {
		panic(2)
	}
	if varargs_overload.Vararg_over2(2, 2.2) != "2 2.2" {
		panic(3)
	}

	if varargs_overload.Vararg_over3("Hello") != "Hello" {
		panic(4)
	}
	if varargs_overload.Vararg_over3(2, 2.2, "hey") != "2 2.2 hey" {
		panic(5)
	}

	if varargs_overload.Vararg_over4("Hello") != "Hello" {
		panic(6)
	}

	if varargs_overload.Vararg_over4(123) != "123" {
		panic(7)
	}

	if varargs_overload.Vararg_over4("Hello", 123) != "Hello" {
		panic(8)
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/varargs_overload_runme.py
import varargs_overload

if varargs_overload.vararg_over1("Hello") != "Hello":
    raise RuntimeError, "Failed"

if varargs_overload.vararg_over1(2) != "2":
    raise RuntimeError, "Failed"


if varargs_overload.vararg_over2("Hello") != "Hello":
    raise RuntimeError, "Failed"

if varargs_overload.vararg_over2(2, 2.2) != "2 2.2":
    raise RuntimeError, "Failed"


if varargs_overload.vararg_over3("Hello") != "Hello":
    raise RuntimeError, "Failed"

if varargs_overload.vararg_over3(2, 2.2, "hey") != "2 2.2 hey":
    raise RuntimeError, "Failed"

if varargs_overload.vararg_over4("Hello") != "Hello":
    raise RuntimeError, "Failed"

if varargs_overload.vararg_over4(123) != "123":
    raise RuntimeError, "Failed"

if varargs_overload.vararg_over4("Hello", 123) != "Hello":
    raise RuntimeError, "Failed"
